### ISSUE TITLE
New ASM installer script requres curl, which is explicitly removed from CFT build images

### ISSUE THREAD

**Issue by matthewkrupnik** [GoogleCloudPlatform/anthos-service-mesh-packages#627]
Current ASM installer script uses curl
https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/0cac367f5dfd945e7cdfd8ad719a688b08acb91b/scripts/asm-installer/install_asm#L1432

Older version was using `kpt` which was being installed as gcloud components so there were no issues. If the new version is being used with CFT builder image, there is no curl installed
https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/3c21b4695f3a48314bab3ff3804e38f853b1afd1/modules/cloudbuild/cloudbuild_builder/Dockerfile#L37

So I keep getting an error saying curl not found and I have to now create custom build images. Are these modules tested with Google's CFT images? What images are being used for testing these modules?

**Comment by zerobfd** [comment]
When you say "older version", you mean the installer that was part of the Terraform module and not the older version in this repo, right? As far as I know we've always had `curl` to get the tarball.

We don't test with the CFT images. (Maybe we should, it would make sense to align, but we're not part of CFT.) The image we use to test the script is here: https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/Dockerfile

Are you using the TF module from here? https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/v14.2.0/modules/asm

**Comment by matthewkrupnik** [comment]
Yeah. We were using version `12.0.0` before and it was using `ktp` to pull all the files it needed. We tried to upgrade to `14.2.0` and suddenly got the curl errors

**Comment by bharathkkb** [comment]
@zerobfd  This should be fixed by https://github.com/terraform-google-modules/terraform-google-bootstrap/pull/106
